paddle/fluid/framework/data_layout_transform.cc

```diff
@@ -88,5 +91,84 @@ void TransDataLayout(const OpKernelType& kernel_type_for_var,
   out->set_layout(expected_kernel_type.data_layout_);
 }
 
+#ifdef PADDLE_WITH_MKLDNN
+using mkldnn::memory;
+using mkldnn::primitive;
+using mkldnn::reorder;
+
+void* get_data_from_tensor(const Tensor& tensor,
```

Comment:
How about `get_data_from_tensor ` be `GetDataFromTensor`? Since we use Camel Case in naming.

Reply:
Done.